.github/workflows/ci.yml

```diff
@@ -15,18 +15,14 @@ jobs:
     - name: Setup Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16
-    - name: Install golint
-      run: |
-        go get -u golang.org/x/lint/golint
+        go-version: 1.18
     - name: Codespell action
       uses: sanketsudake/codespell-action@v1
     - name: Before build
       run: |
         echo Workflow trigger - ${{ github.event_name }}
         ./hack/verify-gofmt.sh
         ./hack/verify-govet.sh
-        ./hack/verify-golint.sh
```

Comment:
@pkosiec Why are we removing this? Won't we miss the lint check coverage?

Reply:
@PrasadG193  golint is deprecated, and today I will introduce PR today with usage of `golangci-lint`: https://github.com/pkosiec/botkube/blob/golangci-lint/.github/workflows/ci.yml#L21.

But you're right - I shouldn't do it right now, but do on the next PR (with golangci-lint). I will revert that change in a moment 👍

Reply:
Done 🙂